public/apps/configuration/panels/audit-logging/audit-logging.tsx

```diff
@@ -61,10 +61,10 @@ function renderStatusPanel(onSwitchChange: () => void, auditLoggingEnabled: bool
             <EuiText color="subdued" grow={false}>
               <FormattedMessage
                 id="audit.logs.storageInstruction"
-                defaultMessage="Configure the output location and storage types in {elasticsearchCode}. The default storage location is {internalElasticsearchCode}, which stores the logs in an index on this cluster."
+                defaultMessage="Configure the output location and storage types in {opensearchCode}. The default storage location is {internalopensearchCode}, which stores the logs in an index on this cluster."
                 values={{
-                  elasticsearchCode: <EuiCode>elasticsearch.yml</EuiCode>,
-                  internalElasticsearchCode: <EuiCode>internal_elasticsearch</EuiCode>,
+                  opensearchCode: <EuiCode>opensearch.yml</EuiCode>,
+                  internalopensearchCode: <EuiCode>internal_opensearch</EuiCode>,
```

Comment:
Minor nit: https://github.com/opensearch-project/opensearch-plugins/blob/main/UPGRADING.md#naming-convention
Please update L64.
```suggestion
                  internalOpenSearchCode: <EuiCode>internal_opensearch</EuiCode>,
```

Reply:
done